python/ray/tune/impl/tuner_internal.py

```diff
@@ -116,14 +151,29 @@ def __init__(
         # without allowing for checkpointing tuner and trainable.
         # Thus this has to happen before tune.run() so that we can have something
         # to restore from.
-        tuner_ckpt = os.path.join(self._experiment_checkpoint_dir, _TUNER_PKL)
-        with open(tuner_ckpt, "wb") as fp:
+        experiment_checkpoint_path = Path(self._experiment_checkpoint_dir)
+        with open(experiment_checkpoint_path / _TUNER_PKL, "wb") as fp:
             pickle.dump(self, fp)
 
-        trainable_ckpt = os.path.join(self._experiment_checkpoint_dir, _TRAINABLE_PKL)
-        with open(trainable_ckpt, "wb") as fp:
+        with open(experiment_checkpoint_path / _TRAINABLE_PKL, "wb") as fp:
             pickle.dump(self._trainable, fp)
 
+    def _maybe_sync_down_tuner_state(self, restore_path: str) -> Tuple[bool, str]:
+        """Sync down trainable state from remote storage.
+
+        Returns:
+            Tuple of (synced, local_dir)
```

Comment:
nit: I feel like `synced` itself is pretty ambiguous, because I would expected something that's already local to be considered as synced.